dev/bots/test.dart

```diff
@@ -345,8 +345,15 @@ Future<void> _runWebToolTests() async {
 Future<void> _runBuildTests() async {
   final List<FileSystemEntity> exampleDirectories = Directory(path.join(flutterRoot, 'examples')).listSync()
     ..add(Directory(path.join(flutterRoot, 'packages', 'integration_test', 'example')))
+    ..add(Directory(path.join(flutterRoot, 'dev', 'integration_tests', 'abstract_method_smoke_test')))
```

Comment:
I think this one (and possibly others) are running on firebase test lab.

Reply:
I totally forgot those run on pre-submit, good call.

Reply:
That's the only one.